FLS01/README.md

```diff
@@ -246,18 +273,22 @@ Response:
 ```
 {
   "order_id": "8ed13c2a-a8c6-4f0e-b43e-3fdbf1f094a6": {
-    "amount_fiat": "1000",
+    "amount_fiat": 1000,
     "currency_id": 1,
     "payment_option_id":1,
     "amount_sats" : 800000 ,
+    "btc_price": 69420
+    "order_fee": 1234
     "order_status": "finished"
     "order_status_date": "2023-09-20T00:25:11.123Z"
   },
     "order_id": "8ed13c2a-a8c6-4f0e-b43e-3fdbf1f0333a": {
-    "amount_fiat": "1000",
+    "amount_fiat": 1000,
```

Comment:
add # unit cents ?

Reply:
those comments are more for added info, its specified at the top what the units are, but sure i can add for clarity

Reply:
Right, didn't check the whole file again, all good then!

Reply:
@ademcan i've added units to everything now